osmosis/src/setupStore.js

```diff
@@ -107,7 +114,10 @@ const setupStore = (useCustomHook, config = {}) => {
     return Wrapper;
   };
 
+  const useStore = () => useContext(StoreContext);
+
   if (!!Proxy && config.proxyEnabled) {
+    storeProxyObject['useStore'] = useStore;
```

Comment:
Returning this from the Proxy in a similar fashion to `Context` and `Provider` makes this difficult/impossible to mock due to the nature of Proxies. Setting it on the ref makes this simple.